node.gyp

```diff
@@ -289,6 +289,9 @@
           '-Wl,-bnoerrmsg',
         ],
       }],
+      ['OS=="linux" or OS=="mac"', {
+        'libraries': ['-latomic'],
```

Comment:
This might be a new build requirement, I'm curious to see whether libatomic actually exists on all our linuxen. /cc @nodejs/build

Reply:
It was already a build requirement for V8, see https://github.com/nodejs/node/issues/27377

Reply:
Also https://github.com/nodejs/build/pull/1763#issuecomment-483427632 😉

Reply:
I think atomics are a requirement, but where they come from depends on the toolset.